### PR TITLE
do not query work group params on gles < 3.1

### PR DESCRIPTION
**Pull request by mrk-its**
**Description**

gles backend queries `MAX_COMPUTE_WORK_GROUP_COUNT` and `MAX_COMPUTE_WORK_GROUP_SIZE` parameters generating `getIndexedParameter: Bad `pname`: Invalid enum value` warnings. This PR disables these queries on wasm32 and sets `max_compute_workgroup_size_*` and `max_compute_workgroups_per_dimension` to 
0

**Testing**
It can be tested on wgpu `cube` example

### CODE REVIEW

File: wgpu-hal/src/gles/adapter.rs [hunk 1]
```diff
@@ -328,11 +328,14 @@ impl super::Adapter {
             gl.get_parameter_i32(glow::MAX_VERTEX_UNIFORM_BLOCKS)
                 .min(gl.get_parameter_i32(glow::MAX_FRAGMENT_UNIFORM_BLOCKS)) as u32;
 
-        let max_compute_workgroups_per_dimension = gl
-            .get_parameter_indexed_i32(glow::MAX_COMPUTE_WORK_GROUP_COUNT, 0)
-            .min(gl.get_parameter_indexed_i32(glow::MAX_COMPUTE_WORK_GROUP_COUNT, 1))
-            .min(gl.get_parameter_indexed_i32(glow::MAX_COMPUTE_WORK_GROUP_COUNT, 2))
-            as u32;
+        let max_compute_workgroups_per_dimension = if cfg!(not(target_arch = "wasm32")) {
```
Comment (kvark) [inline wgpu-hal/src/gles/adapter.rs:331]: I think this check should be for GLES version not WASM32.
Something like this:
```rust
let supports_work_group_params = ver >= (3, 1);
let max_compute_workgroups_per_dimension = if supports_work_group_params {...
...
} else {...};
```

**Review by kvark — CHANGES_REQUESTED** [review]

File: wgpu-hal/src/gles/adapter.rs (hunk 1 above)
Comment (mrk-its) [reply wgpu-hal/src/gles/adapter.rs]: done!

**Review by kvark — APPROVED** [review]
Thank you!

**Comment by kvark** [comment]
Published in wgpu-hal-0.11.5

**Pull request merged**